Source/DafnyDriver/Commands/BuildCommand.cs

```diff
@@ -1,21 +1,30 @@
 using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
 using System.Linq;
 
 namespace Microsoft.Dafny;
 
 class BuildCommand : ICommandSpec {
-  public string Name => "build";
-
-  public string Description => "Produce an executable binary.";
-
   public IEnumerable<IOptionSpec> Options => new IOptionSpec[] {
     OutputOption.Instance,
     TargetOption.Instance,
     NoVerifyOption.Instance,
     CompileVerboseOption.Instance,
   }.Concat(CommandRegistry.CommonOptions);
 
-  public void PostProcess(DafnyOptions dafnyOptions, Options options) {
+  public Command Create() {
+    var result = new Command("build", "Produce an executable binary.");
```

Comment:
did I miss something or it should be "Produce an executable binary or a library"?